packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-network-filter-mobile.tsx

```diff
@@ -1,15 +1,17 @@
 /* eslint-disable camelcase --- ... */
 import { useMyJobsFilterStore } from '@/hooks/use-my-jobs-filter-store';
-import { chains } from '@/smart-contracts/chains';
 import { Filtering } from '@/components/ui/table/table-header-menu.tsx/filtering';
+import { useGetAllNetworks } from '@/hooks/use-get-all-networks';
 
-const allNetworks = chains.map(({ chainId, name }) => ({
-  option: chainId,
-  name,
-}));
+interface MyJobsNetworkFilterMobileProps {
+  chainIdsEnabled: number[] | undefined;
```

Comment:
Here and in other components that rely on `chainIdsEnabled`: its prop type should be always `number[]`, no `undefined`

If somewhere top-level it's `undefined` - then top-level component should deal with it

Reply:
Definitely a better approach, thanks